### ISSUE TITLE
Concourse 7.9.0 semver resource issue to retrieve username/password

### ISSUE THREAD

**Issue by hooppuser** [concourse/concourse#8659]
## Summary

After upgrading Concourse from 7.7.1 to 7.9.0 all our pipelines failed to use new semver resource. For all pipelines with semver git resource we get the following error:
selected worker: aws-worker-1
Cloning into '/tmp/semver-git-repo'...
fatal: could not read Username for 'https://hoopp.visualstudio.com': No such device or address
error checking for new versions: exit status 128

## Steps to reproduce
1) Upgrading Concourse from 7.7.1 to 7.9.0
2) Start Concourse web and worker

## Expected results
Pipelines with semver git resource should properly initialize

## Actual results
selected worker: aws-worker-1
Cloning into '/tmp/semver-git-repo'...
fatal: could not read Username for 'https://hoopp.visualstudio.com': No such device or address
error checking for new versions: exit status 128

## Additional context
It start working when username and password are moved to uri like:
uri: https://((git-username)):((git-password))@hoopp.visualstudio.com/ISG/_git/Risk-InfrastructureServices

## Triaging info

* Concourse version: 7.9.0
* Browser (if applicable): Chrome Version 108.0.5359.125 (Official Build) (64-bit)
* Did this used to work? Worked in 7.7.1

**Comment by jensmartin** [comment]
We are also seeing this same issue

**Comment by cqwense** [comment]
+1

**Comment by zag1024** [comment]
Are you running your wireless in ECS?
I had this issue when using the ECS-optimized Amazon Linux 2 image _and_ the containerd runtime.

There is some kind of compatibility issue between Linux kernel 4, containerd, and Concourse 7.9, but I'm not clear on what exactly that is.

 You can fix this by switching back to guardian (not optimal) or switching the worker images to using the Amazon ECS-optimized Amazon Linux 2 kernel 5.10 AMI. See details at of these images [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html).

**Comment by zag1024** [comment]
> There is some kind of compatibility issue between Linux kernel 4, containerd, and Concourse 7.9, but I'm not clear on what exactly that is.

Reading further issues, this is likely due to changes in cgroups versions. The workaround remains the same!

**Comment by hooppuser** [comment]
> Are you running your wireless in ECS? I had this issue when using the ECS-optimized Amazon Linux 2 image _and_ the containerd runtime.
> 
> There is some kind of compatibility issue between Linux kernel 4, containerd, and Concourse 7.9, but I'm not clear on what exactly that is.
> 
> You can fix this by switching back to guardian (not optimal) or switching the worker images to using the Amazon ECS-optimized Amazon Linux 2 kernel 5.10 AMI. See details at of these images [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html).

No, we use Ubuntu 20.04.3 with kernel version 5.15.0-1030-aws and we don't have plans to move to another Linux version in the near future or switch back to Guardian. 
The main question is in what Concourse release it will be fixed without changing anything on the client side?
We cannot upgrade to the latest Concourse version now due to this problem.

**Comment by zag1024** [comment]
You haven't mentioned where the credentials are currently stored. There are open issues around Vault and 7.9, so you might be stuck if that's what you're using. I've been using the AWS Secret Manager in 7.9.0, which is working.

**Comment by hooppuser** [comment]
> You haven't mentioned where the credentials are currently stored. There are open issues around Vault and 7.9, so you might be stuck if that's what you're using. I've been using the AWS Secret Manager in 7.9.0, which is working.

We store our credentials in AWS Parameter Store. As I mentioned it worked in 7.9.0 if we move credentials into GIT URL itself.
